parser.js

```diff
@@ -99,11 +100,14 @@ function parse(code, options) {
          */
         if (typeof options.loggerFn === "function") {
             extra.log = options.loggerFn;
+        } else if (options.loggerFn === false) {
+            /** */ // eslint-disable-line valid-jsdoc
```

Comment:
This is a bit odd 😄  Why not put the `// eslint-disable-line valid-jsdoc` on the line you are disabling? Alternatively you could use `Function.prototype` as the value of the noop